stactools_planet/stactools/planet/commands.py

```diff
@@ -60,8 +62,16 @@ def planet():
                   '--title',
                   help='Optional title for the STAC collection.')
     @click.option('--skip-validation', help='Skip validation on the STAC')
+    @click.option('-c',
+                  '--catalog-type',
+                  type=click.Choice([
+                                    pystac.CatalogType.ABSOLUTE_PUBLISHED,
+                                    pystac.CatalogType.RELATIVE_PUBLISHED,
+                                    pystac.CatalogType.SELF_CONTAINED
+                                    ],
+                                    case_sensitive=False))
```

Comment:
I was going to suggest just [iterating over the enum](https://stackoverflow.com/a/29503454), like `[e.value for e in pystac.CatalogType]`, but then I learned that `pystac.CatalogType` doesn't subclass from `Enum`. @lossyrob is there a reason you don't want `CatalogType` to subclass from `Enum`, or would you accept a PR? I'd have to double check that it would be backwards compatible but it should be.

Reply:
I see this is https://github.com/stac-utils/pystac/issues/231

Reply:
Notice that there are other places in the stactools cli where that list is used. I actually just took it from a different command that has the option to specify the type of catalog, so in case of having enums available, it should be changed not just here